src/libraries/Microsoft.Extensions.Configuration.Json/src/JsonConfigurationFileParser.cs

```diff
@@ -52,6 +52,28 @@ private void VisitElement(JsonElement element)
                 ExitContext();
             }
 
+            SetNullIfElementIsEmpty(isEmpty);
+        }
+
+        private void VisitArrayElement(JsonElement element)
+        {
+            var isEmpty = true;
+            int index = 0;
+
+            foreach (JsonElement arrayElement in element.EnumerateArray())
+            {
+                isEmpty = false;
```

Comment:
>isEmpty [](http://example.com/codeflow?start=16&length=7)

you don't need this variable. you can just get rid of it and then do `SetNullIfElementIsEmpty(index == 0)` at the end.

Reply:
yes please. It can still be readable if you write the call as `SetNullIfElementIsEmpty(isEmpty: index == 0)`

Reply:
Thats true, okay.